 // Ovaj custom exception kao parametre prima string sa porukom greške i Exception objekat koji sadrži "pravi" razlog problema
// Zamisao ovoga je da korisniku pošaljemu neku generičnu poruku, dok bi developer kroz innnerException znao pravi razlog

public class ExceptionalException : Exception
{
    public ExceptionalException(string message, Exception innerException)
        :base(message, innerException)
    {
    }
}


StefanBalaban DotNet